packages/beeq/src/components/select/scss/bq-select.variables.scss

```diff
@@ -26,31 +26,31 @@
    * @prop --bq-select--text-size - Select text size
    * @prop --bq-select--text-placeholder-color - Select placeholder text color
    */
-  --bq-select--background-color: theme('colors.ui.primary');
+  --bq-select--background-color: theme(colors.ui.primary);
 
-  --bq-select--border-color: theme('colors.stroke.tertiary');
-  --bq-select--border-color-focus: theme('colors.stroke.brand');
-  --bq-select--border-radius: theme('borderRadius.s');
-  --bq-select--border-width: theme('borderWidth.s');
+  --bq-select--border-color: theme(colors.stroke.tertiary);
+  --bq-select--border-color-focus: theme(colors.stroke.brand);
+  --bq-select--border-radius: theme(borderRadius.s);
+  --bq-select--border-width: theme(borderWidth.s);
   --bq-select--border-style: solid;
 
-  --bq-select--gap: theme('spacing.xs');
+  --bq-select--gap: theme(spacing.xs);
 
-  --bq-select--helper-margin-top: theme('spacing.xs');
-  --bq-select--helper-text-size: theme('fontSize.s');
-  --bq-select--helper-text-color: theme('colors.text.primary');
+  --bq-select--helper-margin-top: theme(spacing.xs);
+  --bq-select--helper-text-size: theme(fontSize.s);
+  --bq-select--helper-text-color: theme(colors.text.primary);
 
   --bq-select--icon-size: 24px;
 
-  --bq-select--label-margin-bottom: theme('spacing.xs');
-  --bq-select--label-text-size: theme('fontSize.s');
-  --bq-select--label-text-color: theme('colors.text.primary');
+  --bq-select--label-margin-bottom: theme(spacing.xs);
+  --bq-select--label-text-size: theme(fontSize.s);
+  --bq-select--label-text-color: theme(colors.text.primary);
 
   --bq-select--padding-start: calc(#{theme('spacing.m')} - var(--bq-select--border-width));
```

Comment:
Wondering if this could work here, can we test it?

```suggestion
  --bq-select--padding-start: calc(#{theme(spacing.m)} - var(--bq-select--border-width));
```

Reply:
After some tests in seems that removing the quotes when using the Tailwind's 'theme()' inside SCSS string interpolation .... does not work .... because I got this error:  👇

<img width="1334" alt="Screenshot 2024-05-23 at 21 36 14" src="https://github.com/Endava/BEEQ/assets/98550729/8a090ee0-190e-4527-9171-e42bdeb7dca1">